tests/frontend/onnx/test_type.py

```diff
@@ -1,16 +1,54 @@
+import onnx
 import pytest
 
-from xdsl.dialects.builtin import TensorType, f32, f64
+from xdsl.dialects.builtin import Float32Type, TensorType, f32, f64
 
 try:
     from onnx import TensorShapeProto, TypeProto  # noqa: E402
 
-    from xdsl.frontend.onnx.shape_type import get_shape, get_tensor_type  # noqa: E402
+    from xdsl.frontend.onnx.type import (
+        get_elem_type,  # noqa: E402
```

Comment:
I'm now wondering whether these are necessary. Could you please try removing the `noqa` comments, and seeing what happens?